src/interface/types.rs

```diff
@@ -478,7 +436,6 @@ pub fn get_ioctlptrunion<'a>(union_argument: Arg) -> Result<&'a mut u8, i32> {
     ));
 }
 
-
 // pub fn get_ioctlptrunion(union_argument: Arg) -> Result<IoctlPtrUnion, i32> {
```

Comment:
why do we have commented code here?

Reply:
Yash commented these code 2 month ago.

Reply:
Ok well do we need this? It seems like we'll need something for ioctl. Lets not commit commented code.

Reply:
Just get rid of commented version, theres a working version above

Reply:
Still need to remove this.

Reply:
ok